flang/runtime/extensions.cpp

```diff
@@ -248,5 +249,12 @@ std::int64_t FORTRAN_PROCEDURE_NAME(access)(const char *name,
 }
 #endif
 
+// CHDIR(DIR)
+void RTNAME(Chdir)(const char *dir, int *status) {
+  int stat = chdir(dir);
```

Comment:
I am not sure it should be used on windows, although I am not an expert here: https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/chdir?view=msvc-170